features/f90-mode.feature

```diff
@@ -0,0 +1,97 @@
+Feature: F90 mode basics
+  Background:
+    When the buffer is empty
+    When I turn on f90-mode
+    When I turn on electric-operator-mode
+
+  Scenario: don't modify string literal
+    When I set electric-operator-enable-in-docs to nil
+    When I type "'var+foo-1'"
+    Then I should see "'var+foo-1'"
+
+  Scenario: don't modify string literal after operator
+    When I type "a+''"
+    Then I should see "a + ''"
+
+  ## * and ** operators
+  Scenario: Space *
+    When I type "a*b"
+    Then I should see "a * b"
+
+  Scenario: Multiplication after a function
+    When I type "f(x)*a"
+    Then I should see "f(x) * a"
+
+  Scenario: Exponentiation after a function
+    When I type "f(x)**a"
+    Then I should see "f(x) ** a"
+
+  Scenario: Write statements with default formatting and default output
+    When I type "write(*,*)"
+    Then I should see "write(*, *)"
+
+  Scenario: Write statements with custom formatting and default output
+    When I type "write(*, '(format)')"
+    Then I should see "write(*, '(format)')"
+
+  Scenario: Write statements with custom formatting and default output
```

Comment:
These two descriptions are the same, I guess that's unintended? 

(I'm not sure if ecukes handles duplicate descriptions properly, ecukes is... less robust than you might hope)

Reply:
This was a typo, and has now been fixed. It was good fun getting a look at ecukes, it's a very elegant way to manage test cases!